Makefile

```
@@ -97,6 +97,7 @@ SED ?= $(shell which gsed 2>/dev/null || which sed)
 # in that case.
 %/$(UPTODATE): GOOS=linux
 %/$(UPTODATE): %/Dockerfile
+	if [ -f $(@D)/Dockerfile.distroless ]; then $(SUDO) docker build -f $(@D)/Dockerfile.distroless --build-arg=revision=$(GIT_REVISION) --build-arg=goproxyValue=$(GOPROXY_VALUE) -t $(IMAGE_PREFIX)$(shell basename $(@D)) -t $(IMAGE_PREFIX)$(shell basename $(@D)):distroless-$(IMAGE_TAG) $(@D)/; fi;
```

```suggestion
	if [ -f $(@D)/Dockerfile.distroless ]; then \
		$(SUDO) docker build -f $(@D)/Dockerfile.distroless \
			--build-arg=revision=$(GIT_REVISION) \
			--build-arg=goproxyValue=$(GOPROXY_VALUE) \
			-t $(IMAGE_PREFIX)$(shell basename $(@D)):distroless-$(IMAGE_TAG) \
			$(@D)/; \
	fi
```

In addition to breaking this into lines, I've removed `-t $(IMAGE_PREFIX)$(shell basename $(@D))`, because this tag will be overwritten by the next docker build command.